examples/README.md

```diff
@@ -193,6 +211,59 @@ All examples should be heavily commented and the comments should make sense in t
 
 ## Tests
 
-All examples should be covered by e2e tests to ensure they successfully do the action the code sample is showing in the docs site.
+All examples need to lint and build without failing. If you have lint issues, run `yarn lint --fix` from the root of your example to fix and/or see the problems.
+
+All examples should be covered by basic e2e tests to ensure they at least render the examples. Ideally they would also have e2e tests that prove the functionality that you're trying to show works. Depending on what you're doing in the examples, it may be difficult to e2e test certain things e.g. logging in with Passport. For this reason, testing of functionality with e2e testing is recommended if practical, but not required.
+
+Install `@playwright/test` as a dev dependency for the e2e tests.
+
+```
+yarn add -D @playwright/test
+```
+
+Create a `playwright.config.ts` file in the root of the example app and add this configuration;
+
+```
+import { defineConfig, devices } from "@playwright/test";
+
+export default defineConfig({
+  testDir: "./tests",
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: "html",
+
+  use: {
+    baseURL: "http://localhost:3000",
+    trace: "on-first-retry",
+  },
+
+  projects: [
+    { name: "chromium", use: { ...devices["Desktop Chrome"] } },
+    { name: "firefox", use: { ...devices["Desktop Firefox"] } },
+    { name: "webkit", use: { ...devices["Desktop Safari"] } },
+
+    { name: "Mobile Chrome", use: { ...devices["Pixel 5"] } },
+    { name: "Mobile Safari", use: { ...devices["iPhone 12"] } },
+  ],
+
+  webServer: {
+    command: "yarn dev",
+    url: "http://localhost:3000",
```

Comment:
I know vite uses some other localhost url like localhost:4372 or something like that, so maybe you could mention that this needs to be updated to match the used url?